model/perf.go

```diff
@@ -736,3 +738,20 @@ func (r *PerformanceResults) FindOutdatedRollups(ctx context.Context, name strin
 
 	return errors.WithStack(it.Close(ctx))
 }
+
+// OverrideInfo describes if a patch build should override mainline results,
+// the reason for the override and the person that issued the override.
+// Also stores the information where in the timeseries the patch fits.
+type OverrideInfo struct {
+	OverrideMainline bool   `bson:"override_mainline"`
+	Order            int    `bson:"order,omitempty"`
```

Comment:
Can we make this a bit more explicit? Something like `BaseOrder`.

Reply:
Good catch, having two fields called order in the same db entry seems questionable :)